examples/otel-logs-splunk/docker-compose.yml

```diff
@@ -37,4 +37,15 @@ services:
     depends_on:
       - splunk
     ports:
-      - 18088:8088
\ No newline at end of file
+      - "18088:8088"
+      - "8888:8888"
+  # Prometheus
+  prometheus:
```

Comment:
I'd recommend keeping the otel-logs-splunk example as small as possible, you could create a new example (named something like otel-logs-internal-monitoring) that includes your changes.

Reply:
I was on the fence on that. Will do.